projects/1.18-fabric/assets/textile-backup/textile_backup/lang/zh_cn.json

```
@@ -0,0 +1,60 @@
+{
+  "text.autoconfig.textile_backup.title": "Textile Backup 配置",
+
+  "text.autoconfig.textile_backup.category.default": "一般",
```

```suggestion
  "text.autoconfig.textile_backup.category.default": "通用",
```
（没看游戏内表现，光靠猜的话……这个可能合适）

---

这个就是设置页面选项卡的大标题。“通用”确实更合适一点。